compose/build.gradle.kts

```diff
@@ -105,4 +105,6 @@ dependencies {
     junitDependencies()
     mockitoKotlinDependencies()
     truthDependencies()
+    coreLibraryDesugaringDependencies()
```

Comment:
@IvanTelisman Just wanted to make sure that this `coreLibraryDesugaringDependencies ` addition was intentional

Reply:
Hey @br-jared-stockton, yup this one was intentional I was seeing this error without when I tried to run the tests.

![Screen Shot 2022-09-20 at 3 49 34 PM](https://user-images.githubusercontent.com/84419633/191363308-aa54b4ab-c253-4cd5-a004-a12cc8cdd386.png)